test/helper/test_trigger.ts

```diff
@@ -607,7 +608,7 @@ const testTriggerFile = (file: string) => {
 
         // Responses can have unused output strings in some cases, such as ones
         // that work with and without matching.
-        if (!response && !dynamicOutputStringAccess) {
+        if (!dynamicOutputStringAccess) {
```

Comment:
This logic was faulty and causing a lot of cases to not get tested properly. `(!{}) === false`

Reply:
Wow, thanks for fixing this.